### ISSUE TITLE
1.8.15 regression: caller graph created even though CALLER_GRAPH = NO

### ISSUE THREAD

**Issue by AntoniMarcinek** [doxygen/doxygen#6882]
Hello,

I have the following in my Doxyfile:
```
SOURCE_BROWSER = YES
CALL_GRAPH = NO
CALLER_GRAPH = NO
REFERENCED_BY_RELATION = YES
REFERENCES_RELATION = YES
```

Up to doxygen 1.8.14 I was getting references(ed) lists, but no call(er) graphs, as expected. With doxygen 1.8.15 built from sources (tarball downloaded via doxygen web page) on my Debian 9, I started to get also caller graphs, what almost triples the size of generated html (from 250 MB to 720 MB). I can suppress caller graphs if I set `REFERENCES_RELATION    = NO`.

You can reproduce the problem by unpacking the attachment and running doxygen 1.8.15 in the report/ directory. Then open html/index.html in a browser, use search to go to the ds_prot_all.h file, then look at the description of ds_action_ function.

I think it might be associated with #2314

Best regards,
Antoni
[report.tar.gz](https://github.com/doxygen/doxygen/files/2954589/report.tar.gz)

**Comment by albert-github** [comment]
The problem is a regression due to pull request #6562 

I've just pushed a proposed patch, pull request #6883

**Comment by albert-github** [comment]
Code has been integrated in master on github (please don't close the issue as this will be done at the moment of a release).

**Comment by AntoniMarcinek** [comment]
Wow, that was fast. I confirm that with the current master the problem is fixed. Thank you!

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.8.16.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
